site/content/en/docs/handbook/accessing.md

```diff
@@ -11,6 +11,8 @@ aliases:
 
 There are two major categories of services in Kubernetes:
 
+[publishing-services-service-types](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types)
```

Comment:
this doesnt seem to be relevent, 

it says
"
There are two major categories of services in Kubernetes:

[publishing-services-service-types]
"

Reply:
Hi @medyagh 

I though it may help users to access the kubernetes service type here.

This line has been removed and the PR is rebased on master branch.

Could you kindly review it again?